package.json

```diff
@@ -9,6 +9,9 @@
   "bugs": {
     "url": "https://github.com/Countly/countly-sdk-react-native-bridge/issues"
   },
+  "scripts": {
+    "postinstall": "node ./hooks/createNotificationService.js"
```

Comment:
what does this do?

Reply:
This isn't doing anything at the moment.

I will remove it.